tests/test_planning.py

```diff
@@ -18,33 +20,43 @@
     Predicate, State, STRIPSOperator, Task, Type, _GroundNSRT, _Option
 
 
-@pytest.mark.parametrize("sesame_check_expected_atoms", [True, False])
-def test_sesame_plan(sesame_check_expected_atoms):
+@pytest.mark.parametrize(
+    "sesame_check_expected_atoms,sesame_grounder,expectation",
+    [(True, "naive", does_not_raise()), (False, "naive", does_not_raise()),
+     (True, "fd_translator", does_not_raise()),
+     (True, "not a real grounder", pytest.raises(ValueError))])
+def test_sesame_plan(sesame_check_expected_atoms, sesame_grounder,
```

Comment:
@tomsilver this test provides a neat example of a pattern that we can use elsewhere going forward -- parametrization with some cases raising an exception